src/pages/past.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import { chakra, Box, Flex, useColorModeValue, Link } from "@chakra-ui/react";
+import vvcoding from "../images/vvvcoding.svg";
+import EventHeader from "../components/EventHeader";
+import { render } from "react-dom";
+class Past extends Component {
+  render() {
+    return (
+      <Flex p={30} w="full" alignItems="center" justifyContent="left">
+        <EventHeader />
+
+        <Box
+          as="span"
+          //  bg={useColorModeValue("white", "gray.900")}
+          // mx={{ md: 3 }}
+          display="flex"
+          //maxW="700px"
+          // shadow={{ md: "md" }}
+          // rounded={{ md: "md" }}
+          //padding="30"
+          width="800px"
+          height="305px"
+          borderWidth="2px"
+          borderRadius="lg"
+        >
+          <Box w={{ lg: "45%" }}>
+            <Box
+              h={{ base: 14, sm: "full" }}
+              rounded={{ lg: "lg" }}
+              bgSize="cover"
+              bgImage={vvcoding}
+              width={300}
+              bgSize="contain"
```

Comment:
Duplicate prop is present here.
Are you using eslint ? As I have mentioned in [readme](https://github.com/sam9111/ssn-acm-w/blob/master/README.md)

Reply:
This page should not been commited.My mistake.